### PR TITLE
Add code-review-and-bip324 review to brink/the-bitcoin-development-podcast

### PR DESCRIPTION
**Pull request by markon1-a**
This PR adds [code-review-and-bip324](https://podcasters.spotify.com/pod/show/bitcoinbrink/episodes/Sebastian-Falbesoner-on-Code-Review-and-BIP324-e2909ue) transcript review to the brink/the-bitcoin-development-podcast directory.

### CODE REVIEW

**Review by markon1-a — COMMENTED** [review]
Thanks for the fixes.

**Review by realdezzy — COMMENTED** [review]
Hey @markon1-a,

Thank you so much for your valuable contribution to this project! I apologize for the delay in reviewing your submission; we're actively working on improving our process to ensure quicker responses.

I've made a few changes to the transcript, and I’d appreciate it if you could review them at your convenience.

We look forward to seeing more of your work in the future. Thanks again for your patience and contribution!

File: brink/the-bitcoin-development-podcast/code-review-and-bip324.md [hunk 1]
```diff
@@ -203,47 +208,48 @@ If that option is turned on, they're just really created on the go, whenever a n
...
 Like it would be great if we have more review power.
-And yeah, according to what person's reviews counts, I think it's also kind of a proof of work system in Bitcoin Core, Not specifically regarding me, but in general, if you over the time give review comments that are helpful or are considered helpful by others, then of course you're getting taken more seriously.
-Seriously and what I, I think my review style has been also inspired by other people from the top of my head.
-I could name Ross Janowski who give.
-Very great detailed review comments and that's inspired someone, right?
+And yeah, according to what person's reviews counts, I think it's also kind of a proof of work system in Bitcoin Core.
+Not specifically regarding me, but in general, if you over the time give review comments that are helpful or are considered helpful by others, then of course you're getting taken more seriously.
+I think my review style has been also inspired by other people from the top of my head.
+I could name Russ yanofsky who give very great detailed review comments and that's inspired someone, right?
```
Comment (kouloumos) [inline brink/the-bitcoin-development-podcast/code-review-and-bip324.md]: Potential places that you can look at when trying to identify and fix a name:
- https://btctranscripts.com/speakers
- search for the name on the repo of the project, e.g https://github.com/bitcoin/bitcoin

File: brink/the-bitcoin-development-podcast/code-review-and-bip324.md [hunk 2]
```diff
@@ -372,24 +377,23 @@ You familiarize yourself with the topic by reading the BIP and maybe looking at
...
-I was lucky enough to join a hangout at the last CoreDev meeting in Ireland where Peter gave an introduction to the LibSecP library and he specifically presented the PR 1129, which is the Alligator Swift part.
+What I first did back then still Dhruv was maintaining the PRs.
+So I just first built the main PR that includes all the sub-PRs and ran a node just to also have something available already and playing a bit around what the end product would look like and then iteratively I would look at those sub-PRS and no, I didn't look at the SECP parts until recently It always was appeared very scary for me all those cryptography and math stuff but I also ended up doing a little review there.
+I was lucky enough to join a hangout at the last CoreDev meeting in Ireland where Peter gave an introduction to the `libsecp` library and he specifically presented the PR1129, which is the Alligator Swift part.
 So I got a little deeper into that through that.
```
Comment (kouloumos) [inline brink/the-bitcoin-development-podcast/code-review-and-bip324.md:388]: reviewing transcripts is an opportunity to learn, so what you could here is to go on your favorite search engine and search for "bitcoin libsecp github PR 1129", which would eventually let you to the PR: https://github.com/bitcoin-core/secp256k1/pull/1129

and there you could see that it's "ElligatorSwift"
```suggestion
I was lucky enough to join a hangout at the last CoreDev meeting in Ireland where Peter gave an introduction to the `libsecp` library and he specifically presented the PR1129, which is the ElligatorSwift part.
```

**Review by kouloumos — APPROVED** [review]
Thank you for your contribution, @markon1-a!

This was a solid review overall. Both @realdezzy and I added a few corrections, primarily around technical terms and names. It's important to be precise with those, so feel free to look them up online or ask questions in Discord whenever you're unsure. A helpful resource you can check out is [Bitcoin Core’s documentation](https://bitcoincore.org/en/doc/), where you can quickly search for RPC commands and other relevant details.

Great work—keep refining your reviews, and don’t hesitate to reach out if you need any help!

**Pull request merged**